src/playlist.rs

```diff
@@ -842,9 +842,13 @@ pub struct MediaSegment {
     pub daterange: Option<DateRange>,
     /// `#EXT-`
     pub unknown_tags: Vec<ExtTag>,
+    /// Precision of floating-point values, such as `#EXTINF:<duration>`
+    pub float_precision: usize,
```

Comment:
I don't think its desirable to have this on the MediaSegment as it is not part of the actual data. 

Instead we have a choice here between:
- an extra parameter to `Playlist.write_to(self, writer, opts)`. This could become part of some `WriteOptions` if any other write configuration has to be supported in the future.
- or a global mutable value like `pub static mut FLOAT_PRECISION: Option<usize> = None;`, that way only people who have to deal with faulty parsers can do their thing. Though this requires `unsafe` and is probably not as common in rust? I'm not sure. 
- or something else similar to option 2 that I'm not familiar with

Reply:
The first option sounds less hacky. I'll try that out.

Reply:
Decided to go with a global atomic. Seems like a good compromise between not changing the API / breaking backwards compatibility and not using unsafe code.

The other option is to implement a new `write_to_opts` method, which accepts WriteOpts and calls the old write_to under the hood.